Sources/PathKit.swift

```diff
@@ -227,7 +227,9 @@ extension Path {
   /// - Returns: the last path component
   ///
   public var lastComponent: String {
-    return NSString(string: path).lastPathComponent
+    var component = NSString(string: path).lastPathComponent
+    component.makeContiguousUTF8()
```

Comment:
it's important to call `makeContiguousUTF8` if it's going to be used much later, e.g. for string comparison. it allows swift to use fast paths in the string code if possible